util/pyclient/bft_client.py

```diff
@@ -69,26 +95,26 @@ def __init__(self, config, replicas):
                                        trio.socket.SOCK_DGRAM)
         self.req_seq_num = ReqSeqNum()
         self.client_id = config.id
-        self.replies = dict()
         self.primary = None
         self.reply = None
         self.retries = 0
         self.msgs_sent = 0
-        self.reply_quorum = 2*config.f + config.c + 1
-        self.port = BASE_PORT + 2*self.client_id
+        self.port = BASE_PORT + 2 * self.client_id
         self.sock_bound = False
+        self.replies_manager = rsi.RepliesManager()
```

Comment:
There needs to either be a mechanism to clear replies, or a new `RepliesManager` needs to be used on each request.

Reply:
done